### DISCUSSION TITLE
Partial access to internet

### DISCUSSION THREAD

**Discussion by DW-homeassistant** [mattlongman/Hassio-Access-Point#76] [Ideas]
Hi,
At the moment, I use this Addon to separate domotic devices (that do not need an internet access) and the computers/tablets etc. The issue is that when adding (and seting) a new WiFi device with a smartphone, it is not easy to connect the Access Point if internet connection is not available as Android prefers switching to another access point, I usually have to temporarily enable Internet Access.

So here is a suggestion : would it be possible to define a partial internet access ? For instance, for a given range of IPs, internet access is possible, and for another one, it is forbidden.

**Comment by ROBOT0-VT** [comment]
Hiya! This is a great suggestion, will look into implementing it once I've finished the rest of the networking refactor =D
